.circleci/config.yml

```diff
@@ -205,10 +205,11 @@ jobs:
           when: always
 
       # Linting
-      - run:
-          name: eslint
-          command: yarn lint --format junit --output-file test-results/eslint/eslint.xml
-          when: always
+      # NOTE: This step is temporarily disabled, until we can upgrade the versions of eslint and its plugins that we use.
```

Comment:
Could you create a work item to re-enable the eslint jobs so that we don't lose track of it?

Reply:
I'm going to add it as a criteria for the "Update the ECMA Engines" story.